src/components/organizations/organizations.ts

```diff
@@ -21,13 +21,16 @@ export async function listOrganizations(ctx: IContext, _params: IParameters): Pr
 
   const organizations = await cf.organizations().then(sortOrganizationsByName);
 
+  const user = await account.fetchLoggedInUser(ctx);
```

Comment:
We could do:

```
const organizations, user = await Promise.all(
  cf.organizations().then(sortOrganizationsByName),
  account.fetchLoggedInUser(ctx),
);
```

And save ourselves a bit of time making the requests sequentially.